### PR TITLE
99 prem agreement screen

### PR DESCRIPTION
**Pull request by yishain11**

### CODE REVIEW

File: frontend/src/pages/AgreementPage.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  )[0];
+  const currentCategory = currentAgreement.category.name;
+  const handleClick = (e: any) => {
+    e.preventDefault();
+    console.log('clicked breadcrumb');
+  };
+  return (
+    <Stack direction="column">
+      <Breadcrumbs separator="›" aria-label="breadcrumb">
+        <Link underline="hover" key="1" color="inherit" href="/" onClick={handleClick}>
```
Comment (nadavmoreno) [inline frontend/src/pages/AgreementPage.tsx:22]: @yishain11 match zeplin `font-size` and `font-weight`

File: frontend/src/pages/AgreementPage.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    (agreement) => agreement.id.toString() === agreementId
+  )[0];
+  const currentCategory = currentAgreement.category.name;
+  const handleClick = (e: any) => {
+    e.preventDefault();
+    console.log('clicked breadcrumb');
+  };
+  return (
+    <Stack direction="column">
+      <Breadcrumbs separator="›" aria-label="breadcrumb">
```
Comment (nadavmoreno) [inline frontend/src/pages/AgreementPage.tsx:21]: @yishain11

-  match zeplin seperator icon
-  add bottom border
-  increase x axis padding

File: frontend/src/pages/AgreementPage.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        <Link underline="hover" key="1" color="inherit" href="/" onClick={handleClick}>
+          {currentCategory || 'categoryName'}
+        </Link>
+        <Link underline="hover" key="1" color="inherit" href="/" onClick={handleClick}>
+          <Stack direction="row" gap="5px">
+            <SummarizeOutlinedIcon />
```
Comment (nadavmoreno) [inline frontend/src/pages/AgreementPage.tsx:27]: @yishain11 match zeplin document icon

File: frontend/src/pages/AgreementPage.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            {localStorage.getItem('agreementName') || 'Agreement Name'}
```
Comment (nadavmoreno) [inline frontend/src/pages/AgreementPage.tsx:28]: @yishain11 match zeplin `font-size` and `font-weight`

File: frontend/src/pages/AgreementPage.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      </Breadcrumbs>
+      <Stack direction="column">
+        <Stack
+          direction="row"
+          alignItems="center"
+          justifyContent="space-between"
+          sx={{ margin: '2rem' }}
+        >
+          <Stack direction="row" alignItems="center" justifyContent="space-between">
+            <Typography variant="h1">
```
Comment (nadavmoreno) [inline frontend/src/pages/AgreementPage.tsx:40]: @yishain11 match zeplin `font-size`

File: frontend/src/pages/AgreementPage.tsx [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+              size="small"
+              sx={{ marginLeft: '1rem', backgroundColor: '#4c67f6', fontSize: '0.8rem' }}
+            />
+          </Stack>
+          <Button variant="contained">
+            <VisibilityOutlinedIcon />
+            <Typography variant="body1"> View Agreement</Typography>
+          </Button>
+        </Stack>
+        <Typography sx={{ paddingLeft: '3rem' }} variant="body2">
```
Comment (nadavmoreno) [inline frontend/src/pages/AgreementPage.tsx:54]: @yishain11 match zeplin `font-size`

File: frontend/src/pages/index.ts [hunk 7]
```diff
@@ -3,5 +3,5 @@ import Home from './Home';
 import AllAgreements from './AllAgreements';
 import NewAgreement from './NewAgreement';
 import { Welcome } from './Welcome';
-
-export { Page404, Home, AllAgreements, NewAgreement, Welcome };
+import AgreementPage from './AgreementPage';
+export { AgreementPage, Page404, Home, AllAgreements, NewAgreement, Welcome };
```
Comment (nadavmoreno) [inline frontend/src/pages/index.ts:7]: @yishain11 add an empty line before line 7

**Review by nadavmoreno — CHANGES_REQUESTED** [review]
@yishain11 
fix blank page when entering an agreement with no category attached

File: frontend/src/pages/AgreementPage.tsx (hunk 3 above)
Comment (yishain11) [reply frontend/src/pages/AgreementPage.tsx:27]: Tsur needs to give me an icon/svg/img

File: frontend/src/pages/AgreementPage.tsx (hunk 2 above)
Comment (yishain11) [reply frontend/src/pages/AgreementPage.tsx:21]: done, icon needs to be provided by Tsur

File: frontend/src/pages/AgreementPage.tsx (hunk 4 above)
Comment (yishain11) [reply frontend/src/pages/AgreementPage.tsx:28]: done

File: frontend/src/pages/AgreementPage.tsx (hunk 5 above)
Comment (yishain11) [reply frontend/src/pages/AgreementPage.tsx:40]: done

File: frontend/src/pages/index.ts (hunk 7 above)
Comment (yishain11) [reply frontend/src/pages/index.ts:8]: done

**Review by nadavmoreno — APPROVED** [review]

**Pull request merged**